e2e/orchestrator_ref_grand_parent_task_test.go

```diff
@@ -33,6 +33,7 @@ func testOrchestratorRefTask(executionStream pb.Execution_StreamClient, instance
 						Key: "n1",
 						Type: &process.Process_Node_Task_{
 							Task: &process.Process_Node_Task{
+								RefKey:       "n1",
```

Comment:
Let's use a different name for `key` and `refKey` to avoid false positive in the tests

```suggestion
								RefKey:       "ref1",
```